types/index.d.ts

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+
+type IGridBreakpoints = 'xs' | 'sx' | 'md' | 'lg' | 'xl';
```

Comment:
is this `sx` intentionally or should it be `sm`?

Reply:
Hello @klaasman 

No! This was just a lack of attention! Thank you very much for alerting us!